app/scripts/modules/core/src/managed/StatusBubble.tsx

```diff
@@ -36,16 +38,47 @@ const transitionConfig = {
   config: { mass: 1, tension: 400, friction: 30 },
 };
 
-export const StatusBubble = memo(({ iconName, appearance, size }: IStatusBubbleProps) => {
+export const StatusBubble = memo(({ appearance, iconName, size, quantity }: IStatusBubbleProps) => {
   const transitions = useTransition(iconName, null, transitionConfig);
+  let quantityPill: ReactNode = null;
+  let withQuantity = false;
+
+  // Render quantity text only for size >= small.
+  if (quantity && (size === 'small' || size === 'medium' || size === 'large' || size === 'extraLarge')) {
```

Comment:
small readability tweak: do you care for the idea of pulling the list of sizes out explicitly and then referencing them here via a list lookup? e.g.

```tsx
// up above component near imports
const QUANTITY_SIZES = ['small', 'medium', 'large', 'extraLarge'];

// ...

if (quantity && QUANTITY_SIZES.includes(size)) {
  // ...
}
```